android-exoplayer/src/main/java/com/brentvatne/exoplayer/ExoPlayerCache.java

```diff
@@ -68,14 +68,14 @@ public void run() {
                 final Uri uri = Uri.parse(url);
                 final DataSpec dataSpec = new DataSpec(uri, 0, C.LENGTH_UNSET, null);
                 final SimpleCache downloadCache = ExoPlayerCache.getInstance(getReactApplicationContext());
-                CacheKeyFactory cacheKeyFactory = ds -> CACHE_KEY_PREFIX + "." + CacheUtil.generateKey(ds.uri);;
+                // CacheKeyFactory cacheKeyFactory = ds -> CACHE_KEY_PREFIX + "." + CacheUtil.generateKey(ds.uri);;
 
                 try {
-                    CacheUtil.getCached(
-                        dataSpec,
-                        downloadCache,
-                        cacheKeyFactory
-                    );
+                    // CacheUtil.getCached(
+                    //     dataSpec,
+                    //     downloadCache,
+                    //     cacheKeyFactory
+                    // );
```

Comment:
Looking at the [source of this method](https://github.com/google/ExoPlayer/blob/r2.10.5/library/core/src/main/java/com/google/android/exoplayer2/upstream/cache/CacheUtil.java#L78) I don't think it actually does something 🤷🏼‍♂️

Reply:
https://relivehq.slack.com/archives/C03FH914E2V/p1652358818112919